opendevin/llm/llm.py

```diff
@@ -95,6 +98,9 @@ def __init__(
         if self.config.drop_params:
             litellm.drop_params = self.config.drop_params
 
+        # Set up config attributes with default values to prevent AttributeError
+        LLMConfig.set_missing_attributes(self.config)
```

Comment:
Can you move this up a bit, so that we can get rid of things like `not hasattr(config, 'max_input_tokens')`